### DISCUSSION TITLE
关于委托过滤器优先级的疑问

### DISCUSSION THREAD

**Discussion by HCRHarry** [LmeSzinc/AzurLaneAutoScript#1461] [Q&A] [answered]
在我使用委托的委托过滤器功能时，发现如下所设置的研究顺序并不能如我所想的执行。

>DailyEvent
> Gem-8 > Gem-4 > Gem-2   

> DailyCube-1 > DailyCube-2
> ExtraCube-0:30 > UrgentCube-1:30 > UrgentCube-1:45 > UrgentCube-2:15 > UrgentCube-3 > ExtraCube-1:30 > ExtraCube-3 
> UrgentCube-4 > UrgentCube-6 > ExtraCube-4 > ExtraCube-5 > ExtraCube-8 
> NightCube-6 > NightCube-7 > NightCube-8

> DailyDrill-1 > DailyDrill-2
> ExtraDrill-1 > ExtraDrill-2 > ExtraDrill-3 > ExtraDrill-3:20 > ExtraDrill-5:20
> NightDrill-8 > NightDrill-7 > NightDrill-6

> DailyChip > DailyResource > DailyBook > DailyBox 

> UrgentBook-2 > UrgentBook-2:30 

> NightOil-8
> UrgentBox-6 > UrgentBox-3 > UrgentBox-1
> shortest
--------------------------------------------------------------------

- 我希望的优先级是：红尖尖>魔方>钻头>图纸>技能书>箱子>其他。
- 我发现如以上设置，会优先执行“ DailyChip > DailyResource ”的分类，即“高阶战术研发>日常资源开发”，再去执行其他的分类。请问各位大佬，是否有优先级设置上的规则是我没有注意到的？或者如何修改来实现以上的期望优先顺序呢？

万分感谢。_(:3⌒ﾞ)_

**Comment by LmeSzinc** [comment, marked as answer]
每日委托只有10个，并且都是收益非常高的，这10个哪个先做哪个后做都没有关系，都是必须要做完，所以阿拉斯里面没有再细分了。

**Reply by HCRHarry** [reply]
好滴明白了，感谢大佬的回答…
